core/src/main/scala/org/apache/spark/SecurityManager.scala

```diff
@@ -81,7 +81,7 @@ import org.apache.spark.util.Utils
  *  - HTTP for broadcast and file server (via HttpServer) ->  Spark currently uses Jetty
  *            for the HttpServer. Jetty supports multiple authentication mechanisms -
  *            Basic, Digest, Form, Spengo, etc. It also supports multiple different login
- *            services - Hash, JAAS, Spnego, JDBC, etc.  Spark currently uses the HashLoginService
+ *            services - Hash, JAAS, Spengo, JDBC, etc.  Spark currently uses the HashLoginService
```

Comment:
The above line had Spengo, so I happened to see it and change it, I might have to change the other way round